modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -71,10 +71,15 @@ func init() {
 // modified the last component of the path (the filename).
 //
 // This handler sets the Etag and Last-Modified headers for static files.
+// Files with an mtime of 1 second past epoch do not get an Etag.
 // It does not perform MIME sniffing to determine Content-Type based on
 // contents, but does use the extension (if known); see the Go docs for
 // details: https://pkg.go.dev/mime#TypeByExtension
 //
+// Variables are set on request context to store the path of the file being
+// served (if serving a precompressed file, its name will be here), and a
+// fs.FileInfo struct with stat data.
+//
```

Comment:
Might be useful to describe what the variables are actually named -- and since this is a terminal handler (it writes the response and does not call a next middleware), the variable can only be used in response wrappers (i.e. returning back "up" the middleware chain). Not sure how to word that exactly but I can offer a suggestion if you'd like :D